tests/Doctrine/Tests/DBAL/Functional/Driver/OCI8/StatementTest.php

```diff
@@ -39,6 +39,23 @@ public function testQueryConversion(string $query, array $params, array $expecte
         );
     }
 
+    /**
+     * @param mixed[] $params
+     * @param mixed[] $expected
+     *
+     * @dataProvider queryConversionProvider
+     */
+    public function testQueryPrepare(string $query, array $params, array $expected) : void
+    {
+        $stmt = $this->connection->prepare($query);
+        $stmt->execute($params);
```

Comment:
Why does the test named `testQueryPrepare()` prepares _and executes_ the query? How is it different from the one that already does `executeQuery()`?

Reply:
`connection->executeQuery` is a high-level method that works correctly with both `named` and `positioned` parameters. In "testQueryPrepare" - oci8statement is prepared, parameters are bound and then it is executed. Here just also there was an error with `named` parameters though Oracle just only supports them

Reply:
I renamed the name of the test to 'testStatementBindParameters' for a better understanding of what is happening